plugins/container/src/caps/parse/parse.cpp

```diff
@@ -192,7 +195,20 @@ bool my_plugin::parse_exit_process_event(
                 m_logger.log(fmt::format("Removing container from procexit: {}",
                                          container_id),
                              falcosecurity::_internal::SS_PLUGIN_LOG_SEV_TRACE);
+#ifdef _HAS_ASYNC
```

Comment:
Improvement on top of #894 : instead of immediately removing the container cache entry without notification, send the `container_removed` event and let our default parsing logic (the same used for events generated by the go-worker) kick in to manage the removal for us.
This way, other plugins can parse the `container_removed` events for rocket, lxc and libvirt-lxc too.

Reply:
> always rely upon procexit event parsing to emit container_removed event thus dropping the removed listeners on go-worker SDKs

I chose this one, that is more coherent and simple.